recipe/scripts/activate.sh

```diff
@@ -0,0 +1 @@
+export JULIA_DEPOT_PATH="${JULIA_DEPOT_PATH}:${PREFIX}/share/SymbolicRegression.jl/depot"
```

Comment:
Appending the depot is not going to do anything if we do not put anything there.

```
(pysr) $ ls share/julia
base  base.cache  environments  julia-config.jl  logs  stdlib  test

(pysr) $ ls share/SymbolicRegression.jl/depot
artifacts  conda  environments  packages
```

The one thing we may need to do is add PyCall.jl to the environment. That is 

```julia
julia -e "using Pkg; Pkg.add(\"PyCall\")"
```

This should not download anything. It should just add PyCall to Project.toml.

Reply:
Maybe we should add PyCall.jl to the `@pysr-0.10.1` environment instead of the default one named after the conda environment.